facia-tool/public/js/widgets/columns/fronts-config.html

```diff
@@ -200,6 +200,10 @@
             <input type="text" data-bind="
                 value: meta.href"/>
 
+            <label>Description</label>
```

Comment:
Just copying and pasting the bit of #9550 that got overridden by refactor.

To answer your question, yes, it would benefit, but none of the labels are correct in that file. I'll fix in a separate issue